ccb.go

```diff
@@ -0,0 +1,299 @@
+				Active  string `xml:"active,omitempty" json:"active,omitempty"`
+				Creator *struct {
+					ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+				} `xml:"creator,omitempty" json:"creator,omitempty"`
+				Modifier *struct {
+					ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+				} `xml:"modifier,omitempty" json:"modifier,omitempty"`
+				Created                   string `xml:"created,omitempty" json:"created,omitempty"`
+				Modified                  string `xml:"modified,omitempty" json:"modified,omitempty"`
+				UserDefinedTextFields     string `xml:"user_defined_text_fields,omitempty" json:"user_defined_text_fields,omitempty"`
+				UserDefinedDateFields     string `xml:"user_defined_date_fields,omitempty" json:"user_defined_date_fields,omitempty"`
+				UserDefinedPulldownFields string `xml:"user_defined_pulldown_fields,omitempty" json:"user_defined_pulldown_fields,omitempty"`
+			} `xml:"individual,omitempty" json:"individual,omitempty"`
+		} `xml:"individuals,omitempty" json:"individuals,omitempty"`
+		FormResponses *struct {
+			Count        string `xml:"count,attr,omitempty" json:"count,omitempty"`
+			FormResponse []*struct {
+				ID   string `xml:"id,attr,omitempty" json:"id,omitempty"`
+				Form *struct {
+					ID string `xml:"id,attr" json:"id,omitempty"`
+				} `xml:"form,omitempty" json:"form,omitempty"`
+				Individual *struct {
+					Name string `xml:",chardata" json:"name,omitempty"`
+					ID   string `xml:"id,attr,omitempty" json:"id,omitempty"`
+				} `xml:"individual,omitempty" json:"individual,omitempty"`
+				Created       string `xml:"created,omitempty" json:"created,omitempty"`
+				Modified      string `xml:"modified,omitempty" json:"modified,omitempty"`
+				ProfileFields *struct {
+					ProfileInfo []*struct {
+						Name string `xml:"name,attr,omitempty" json:"name,omitempty"`
+						Text string `xml:",chardata" json:"text,omitempty"`
+					} `xml:"profile_info,omitempty" json:"profile_info,omitempty"`
+				} `xml:"profile_fields,omitempty" json:"profile_fields,omitempty"`
+				Answers *struct {
+					Title  []string `xml:"title,omitempty" json:"title,omitempty"`
+					Choice []string `xml:"choice,omitempty" json:"choice,omitempty"`
+				} `xml:"answers,omitempty" json:"answers,omitempty"`
+				PaymentInfo string `xml:"payment_info,omitempty"  json:"payment_info,omitempty"`
+			} `xml:"form_response,omitempty" json:"form_response,omitempty"`
+		} `xml:"form_responses,omitempty" json:"form_responses,omitempty"`
+	} `xml:"response,omitempty" json:"response,omitempty"`
+}
+
+type FormResponses struct {
+	Count     string      `json:"count,omitempty"`
+	Responses []*FormData `json:"responses,omitempty"`
+}
+
+type FormData struct {
+	ID          string            `json:"id,omitempty"`
+	ProfileInfo map[string]string `json:"profile_info,omitempty"`
+	Answers     map[string]string `json:"answers,omitempty"`
+	Created     string            `json:"created,omitempty"`
```

Comment:
What about using date time stamps instead? they'll be easier to work with later, and should auto serialize to a string when using json.